backend/src/data/migrations/20220816104044_add_courses_image_path.ts

```diff
@@ -0,0 +1,19 @@
+import { Knex } from 'knex';
+
+const TABLE_NAME = 'courses';
+
+const COLUMN_NAME = 'image_url';
+
+async function up(knex: Knex): Promise<void> {
+  await knex.schema.alterTable(TABLE_NAME, (table) => {
+    table.string(COLUMN_NAME).notNullable();
+  });
+}
+
+async function down(knex: Knex): Promise<void> {
+  await knex.schema.table(TABLE_NAME, (table) => {
```

Comment:
As far as I remember, here should be knex.schema.alterTable as well as in "up" method

Reply:
Following the `knex` docs it looks like `.table` and `.alterTable` are actually the same thing (`.alterTable` is an alias on `.table`) and that's strange - maybe `.alterTable` is more explicit semantically but that's the only difference 🤔

Reply:
I also checked it with some other migrations and it was `.table` there
Gotta research the difference between them, not really sure about it

Reply:
```
if (method === 'table') method = 'alterTable'; 
```
Found an interesting line in `knex` docs :smile:

Reply:
I've talked actually about that 😅